share/jupyterhub/templates/login.html

```diff
@@ -64,6 +64,15 @@ <h1>Sign in</h1>
       id="password_input"
       tabindex="2"
     />
+    <label for='otp_input'>OTP (if required):</label>
```

Comment:
Nice! Can we put this behind an option, so OTP isn't presented as an input if it's not going to be expected? I think that might be confusing for some. Maybe `Authenticator.expect_otp`?